app/javascript/dashboard/store/modules/conversations/actions.js

```diff
@@ -78,7 +78,7 @@ const actions = {
         id: data.conversationId,
         data: payload,
       });
-      if (payload.length < 20) {
+      if (!payload.length) {
```

Comment:
To support the last page messages. If after and before has <20 messages in between, this check would prevent previous messages.